src/uucore/build.rs

```diff
@@ -269,3 +267,50 @@ fn embed_static_utility_locales(
 
     Ok(())
 }
+
+/// Determines which locales to embed into the binary.
+///
+/// To support localized messages in installed binaries (e.g., via `cargo install`),
+/// this function identifies the user's current locale from the `LANG` environment
+/// variable.
+///
+/// It always includes "en-US" to ensure that a fallback is available if the
+/// system locale's translation file is missing or if `LANG` is not set.
+fn get_locales_to_embed() -> Vec<String> {
+    let mut locales = vec!["en-US".to_string()];
```

Comment:
I think we will have only 2 values
a vec is a bit overkill

+ it would be nice to have unit tests for this function

esp with errors